### PR TITLE
Made suicide a config flag

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

Allows disabling of suicide in the config

## Why It's Good For The Game

Suicide verb's kinda banbait at this point. Let's see if we can go without.

## Changelog
:cl:
config: Added suicide to the config.
/:cl:

### CODE REVIEW

**Comment by BlueWildrose** [comment]
Should make it so suicide is enabled during EORG and nothing else.

File: code/modules/client/verbs/suicide.dm [hunk 1]
```diff
@@ -207,6 +207,8 @@
 	message_admins("[key_name(src)] (job: [src.job ? "[src.job]" : "None"]) [is_special_character(src) ? "(ANTAG!) " : ""][ghosting ? "ghosted" : "committed suicide"] at [AREACOORD(src)].")
 
 /mob/living/proc/canSuicide()
+	if(!CONFIG_GET(flag/suicide_allowed))
```
Comment (Ghommie) [inline code/modules/client/verbs/suicide.dm:210]: Little feedback message saying suicide is not enabled/allowed please.

File: config/game_options.txt [hunk 2]
```diff
@@ -597,3 +597,6 @@ MODETIER_VOTING
 
 ## Number of modes dropped by the modetier vote during mode selection, after vote.
 DROPPED_MODES 3
+
+## Whether the suicide verb is allowed.
+# SUICIDE_ALLOWED
```
Comment (Ghommie) [inline config/game_options.txt:602]: Fun police hates gloomy sunday rounds.

**Review by Ghommie — CHANGES_REQUESTED** [review]
You should either add a ticker check or a second config to enable suicides on round end. Perhaps also enable it for VR, but that might warrant some other checks to stop admins from getting null "ckey/dude commited suicide" messages for VR suicides.

File: config/game_options.txt (hunk 2 above)
Comment (Putnam3145) [reply config/game_options.txt:602]: i know what a "gloomy sunday" is but not a "gloomy sunday round"

Comment (Ghommie) [reply config/game_options.txt:602]: Sad.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**